Decred Wallet/Extensions/UIColor.swift

```diff
@@ -10,27 +10,43 @@ import Foundation
 import UIKit
 
 extension UIColor {
-    // todo: significantly fewer colors should be left here once the implementation of the new
-    // interface design is completed as the new design uses fewer colors generally.
     struct appColors {
-        // decred colors are from https://decred.org/brand/
-        static let decredBlue = UIColor.init(hex: "#2970FF")
-        static let decredGreen = UIColor.init(hex: "#41BF53")
-        static let decredOrange = UIColor.init(hex: "#ED6D47")
-        
         static let darkYellowWarning = UIColor.init(hex: "#E7C659")
-        static let green = UIColor.init(hex: "#2DD8A3")
-        static let lightGray = UIColor.init(hex: "#DEE1E3")
-        static let lighterGray = UIColor.init(hex: "#C4CBD2")
-        static let offWhite = UIColor(hex:"#F3F5F6")
-        static let lightOffWhite = UIColor(hex: "#F9FBFA")
-        static let yellowWarning = UIColor.init(hex: "#FFC84E")
+        static let _lightGray = UIColor.init(hex: "#DEE1E3") // temporary name, pending when it will be made obsolete and removed
         static let thinGray = UIColor.init(hex: "#a4abb1")
-        static let darkGray = UIColor.init(hex: "#4e5f70")
+        static let darkerGray = UIColor.init(hex: "#4e5f70")
         static let transparentThinGray = UIColor.init(hex: "#a4abb1", alpha: 0.3)
```

Comment:
Grays should be grouped like this:
```suggestion
        static let gray = UIColor.init(hex: "#E6EAED")
        static let darkGray = UIColor.init(hex: "#C4CBD2")
        static let darkerGray = UIColor.init(hex: "#4e5f70")
        static let lightGray = UIColor.init(hex: "#EDEFF1")
        static let _lightGray = UIColor.init(hex: "#DEE1E3") // temporary name, pending when it will be made obsolete and removed
        static let thinGray = UIColor.init(hex: "#a4abb1")
        static let transparentThinGray = UIColor.init(hex: "#a4abb1", alpha: 0.3)
```

Reply:
`thinGray` and `transparentThinGray` are not part of the new mockup colors, so I'm keeping them in a separate section so we don't forget to make them obsolete as we continue implementing the new mockup pages.

Reply:
Alright